internal/fuse/dir.go

```diff
@@ -20,10 +21,12 @@ var _ = fs.NodeStringLookuper(&dir{})
 
 type dir struct {
 	root        *Root
-	items       map[string]*restic.Node
 	inode       uint64
 	parentInode uint64
 	node        *restic.Node
+	isOpen      bool
```

Comment:
`isOpen` seems redundant. Afaict `isOpen` == `items != nil`

Reply:
Ah - thanks for the hint! Changed it.